src/pages/miyeon/Main/components/feedComment/Comment.js

```diff
@@ -0,0 +1,96 @@
+import { Component } from 'react';
+import UploadComment from './UploadComment';
+
+class Comment extends Component {
+  constructor() {
+    super();
+    this.state = {
+      inputVal: '',
+      commentList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/miyeon/data/mainFeed.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data =>
+        this.setState({
+          commentList: data,
+        })
+      );
+  }
+
+  handleInput = e => {
+    this.setState({
+      inputVal: e.target.value,
+    });
+  };
+
+  addMyComment = e => {
+    e.preventDefault();
+    const { commentList, inputVal } = this.state;
+    const newArr = [
+      ...commentList,
+      {
+        id: commentList.length + 1,
+        name: 'buzzi_nyang',
+        content: inputVal,
+      },
+    ];
+    this.setState({
+      commentList: newArr,
+      inputVal: '', // state 값만 빈값으로 되고, input창의 value 텍스트는 지워지지 않았음
+    });
+    document.getElementsByClassName('comment')[0].value = '';
```

Comment:
```suggestion
    e.target[0].value = '';
```
저도 form으로 구현 했는데 이런 식으로 구현했어요. 혹시 된다면 가져온 인자로 처리하는게 더 효율적 일 것 같아요